Source/Core/Common/GL/GLExtensions/GLExtensions.cpp

```diff
@@ -2198,7 +2198,7 @@ static void InitExtensionList(GLContext* context)
     default:
     case 450:
     {
-      std::string gl450exts[] = {
+      static const char *const gl450exts[] = {
```

Comment:
`s_extension_list` doesn't have a transparent comparator, so I think `s_extension_list[it]` still instantiates a `std::string` out of `it`. IMO you should definitely check if this is the case, as with a proper transparent comparator you could avoid constructing strings everywhere but when an entry is added to the map.